src/Datadog.Trace.ClrProfiler.Managed/Integrations/StackExchange.Redis/ConnectionMultiplexer.cs

```diff
@@ -24,12 +24,16 @@ public static class ConnectionMultiplexer
             Integration = IntegrationName,
             CallerAssembly = "StackExchange.Redis",
             TargetAssembly = "StackExchange.Redis",
-            TargetType = "StackExchange.Redis.ConnectionMultiplexer")]
+            TargetType = "StackExchange.Redis.ConnectionMultiplexer",
+            TargetMinimumVersion = Major1,
+            TargetMaximumVersion = Major1)]
```

Comment:
Maximum version should be "2". `StackExchange.Redis` is up to `2.0.601` and afaik the integration works with 2.x.